Android/FirebaseJobDispatcher/source/Firebase.JobDispatcher/Firebase.JobDispatcher.csproj

```diff
@@ -0,0 +1,112 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Xamarin.Build.Download.0.4.2\build\Xamarin.Build.Download.props" Condition="Exists('..\packages\Xamarin.Build.Download.0.4.2\build\Xamarin.Build.Download.props')" />
```

Comment:
0.4.2 here but 0.4.3 in packages.config, this should probably be updated to match...  There's a couple other cases in this project file of the same (Lines 4 and 100)